toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/WorldScaleSceneViewScope.kt

```diff
@@ -105,4 +184,293 @@ public class WorldScaleSceneViewScope internal constructor(private val sceneView
+     */
+    @Composable
+    internal fun CalibrationViewInternal(
+        onDismiss: () -> Unit,
+        modifier: Modifier = Modifier,
+        colorScheme: WorldScaleCalibrationViewColorScheme = WorldScaleCalibrationViewDefaults.colorScheme(),
+        typography: WorldScaleCalibrationViewTypography = WorldScaleCalibrationViewDefaults.typography(),
+        onHeadingChange: (Double) -> Unit,
+        onElevationChange: (Double) -> Unit,
+        onHeadingReset: () -> Unit,
+        onElevationReset: () -> Unit
+    ) {
+        CompositionLocalProvider(
+            LocalColorScheme provides colorScheme,
+            LocalTypography provides typography
+        ) {
+            Card(
+                modifier = modifier,
+                colors = CardDefaults.cardColors(
+                    containerColor = LocalColorScheme.current.backgroundColor,
+                )
+            ) {
+                Column(
+                    modifier = Modifier.padding(5.dp),
+                ) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        verticalAlignment = Alignment.CenterVertically,
+                        horizontalArrangement = Arrangement.End
+                    ) {
+                        FilledIconButton(
+                            onClick = onDismiss,
+                            modifier = Modifier
+                                .size(36.dp)
+                                .padding(bottom = 4.dp),
+                            colors = IconButtonDefaults.filledIconButtonColors(
+                                containerColor = LocalColorScheme.current.buttonContainerColor,
+                                contentColor = LocalColorScheme.current.buttonContentColor
+                            )
+                        ) {
+                            Icon(
+                                Icons.Filled.Close,
+                                tint = LocalColorScheme.current.buttonContentColor,
+                                contentDescription = "Close Calibration"
+                            )
+                        }
+                    }
+                    // heading
+                    Card(
+                        colors = CardDefaults.cardColors(
+                            containerColor = LocalColorScheme.current.containerColor,
+                        )
+                    ) {
+
+                        JoysliderBar(
+                            title = "Heading",
+                            value = heading,
+                            valueFormat = DecimalFormat("+0.#º ; -#.#º"),
+                            minusContentDescription = "Decrease Heading",
+                            plusContentDescription = "Increase Heading",
+                            resetContentDescription = "Reset Heading",
+                            onMinusClick = {
+                                onHeadingChange(-1.0)
+                                heading -= 1
```

Comment:
Best to reverse the order here, i.e. first set the value, then invoke the callback. Also applies to the other places but will only mention here.
```suggestion
                                heading -= 1
                                onHeadingChange(-1.0)
```